### PR TITLE
Adding roundtripping test for CancelInvocation message

### PR DESCRIPTION
**Pull request by moozzyk**

### CODE REVIEW

File: test/Microsoft.AspNetCore.SignalR.Common.Tests/Internal/Protocol/MessagePackHubProtocolTests.cs [hunk 1]
```diff
@@ -43,6 +43,8 @@ public class MessagePackHubProtocolTests
             new object[] { new[] { new StreamItemMessage("xyz", new CustomObject()) } },
             new object[] { new[] { new StreamItemMessage("xyz", new[] { new CustomObject(), new CustomObject() }) } },
 
+            new object[] { new[] { new CancelInvocationMessage("xyz ") } },
```
Comment (BrennanConroy) [inline test/Microsoft.AspNetCore.SignalR.Common.Tests/Internal/Protocol/MessagePackHubProtocolTests.cs]: space in id needed?

File: test/Microsoft.AspNetCore.SignalR.Common.Tests/Internal/Protocol/TestHubMessageEqualityComparer.cs [hunk 2]
```diff
@@ -47,6 +47,11 @@ private bool InvocationMessagesEqual(HubMessage x, HubMessage y)
                 left.NonBlocking == right.NonBlocking;
         }
 
+        private bool CancelInvocationMessagesEqual(HubMessage x, HubMessage y)
+        {
+            return x is CancelInvocationMessage && y is CancelInvocationMessage;
```
Comment (BrennanConroy) [inline test/Microsoft.AspNetCore.SignalR.Common.Tests/Internal/Protocol/TestHubMessageEqualityComparer.cs]: Shouldn't this check the invocation ids are the same?

File: test/Microsoft.AspNetCore.SignalR.Common.Tests/Internal/Protocol/MessagePackHubProtocolTests.cs (hunk 1 above)
Comment (moozzyk) [reply test/Microsoft.AspNetCore.SignalR.Common.Tests/Internal/Protocol/MessagePackHubProtocolTests.cs]: this is just another character in the invocation id... 

image

File: test/Microsoft.AspNetCore.SignalR.Common.Tests/Internal/Protocol/TestHubMessageEqualityComparer.cs (hunk 2 above)
Comment (moozzyk) [reply test/Microsoft.AspNetCore.SignalR.Common.Tests/Internal/Protocol/TestHubMessageEqualityComparer.cs]: `invocationId` is common to `HubMessage` and is being checked in `Equals`.

**Pull request merged**